lobby/src/main/java/org/triplea/lobby/server/db/BadWordController.java

```diff
@@ -31,17 +29,27 @@ public void addBadWord(final String word) {
   }
 
   @Override
-  public List<String> list() {
-    final String sql = "select word from bad_words";
+  public boolean containsBadWord(final String testString) {
+    if (testString.isEmpty()) {
+      return false;
+    }
+
+    // Query to count if at least one bad word value is contained in the testString.
+    final String sql =
+        "select count(bw.word) "
+            + "from bad_words bw "
+            + "where ? like '%' || lower(bw.word) || '%' "
+            + "limit 1";
```

Comment:
Limit is redundant because count only returns a single line

Reply:
Yeah, I was thinking of an inner query where this would have helped. Does strike me as a premature optimization at the moment.

Updated: f5bc14daf